### ISSUE TITLE
手动支持一下。

### ISSUE THREAD

**Issue by wgqhandsome** [tjmoon0104/pytorch-tiny-imagenet#7]
有几个问题想请教一下。
1. 训练时间。resnet18使用tiny-imagenet数据集训练一轮的时间大概是多久？
2. 关于预训练权重。如果不使用预训练权重的结果大概是多少呢？

**Comment by tjmoon0104** [comment]
There are a couple questions I'd like to ask.

1. training time. what is the approximate time for a training round of resnet18 using the tiny-imagenet dataset?

From my M1 Pro, it took about 6-7 mintues per epoch.

2. about pre-training weights. What is the approximate result if pre-training weights are not used?

10% accuracy in case of no pretrained weight with 10 epoch.

You can see the details [here](https://github.com/tjmoon0104/pytorch-tiny-imagenet/blob/master/ResNet18_64.ipynb)

**Comment by wgqhandsome** [comment]
Thank you very much for your answer. I am very interested in this project of yours and I will continue to follow your project closely.发自我的 iPhone在 2023年12月21日，15:37，TaeJun Moon ***@***.***> 写道：﻿
There are a couple questions I'd like to ask.

training time. what is the approximate time for a training round of resnet18 using the tiny-imagenet dataset?

From my M1 Pro, it took about 6-7 mintues per epoch.

about pre-training weights. What is the approximate result if pre-training weights are not used?

10% accuracy in case of no pretrained weight with 10 epoch.
You can see the details here
